lms/migrations/versions/16ad8b08e0dd_lmscourse_copied_from_id_backfill.py

```diff
@@ -0,0 +1,38 @@
+"""LMSCourse.copied_from_id backfill."""
```

Comment:
There's going to be a few of this type of migration, backfill the info from existing tables.